.travis.yml

```diff
@@ -1,2 +1,6 @@
+notifications:
+  email: false
+
 rvm:
-  - 2.1.0
+  - 2.1.8
+  - 2.2.4
```

Comment:
Indeed, I'd be very interested in upgrading the deployment environments to ruby 2.1.8 or 2.2.4. I'd willingly add 2.1.2, if you think it'd be appropriate (although I'd hope the difference between 2.1.x versions will be very minor)